### PR TITLE
Trigger onViewUpdate on changes in scale and translate

### PR DESCRIPTION
**Pull request by oliveira-mauricio**

### CODE REVIEW

**Review by nicoddemus — APPROVED** [review]

File: environment.devenv.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ dependencies:
     - cherrypy
     - hypothesis ==3.11.0
     - pytest-mock
-    - pytest-qt
+    - pytest-qt==3.3.0
```
Comment (oliveira-mauricio) [inline environment.devenv.yml:26]: Without pin it was latest pytest-qt (4.0.1) which does not support our Qt version:

```ERROR: PyQt < 5.11 is not supported by pytest-qt, use 5.11+ instead.``` 

But this pin should be removed once we upgrade Qt to 5.12.

Comment (nicoddemus) [reply environment.devenv.yml:26]: Yeah I noticed but didn't comment, but I already updated to the latest pytest-qt on my Qt 5.12 branch. 👍

File: qmxgraph/page/api.js [hunk 2]
```diff
@@ -1175,6 +1175,10 @@ graphs.Api.prototype.onViewUpdate = function onViewUpdate (handler) {
     var listener = (function(sender, evt) {
         handler(this.dump(), this.getScaleAndTranslation());
     }).bind(this);
+    graph.getView().addListener(mxEvent.SCALE, listener);
```
Comment (prusse-martin) [inline qmxgraph/page/api.js:1178]: Did you check if those overlap  with any undo event?

**Review by prusse-martin — APPROVED** [review]

File: qmxgraph/page/api.js (hunk 2 above)
Comment (oliveira-mauricio) [reply qmxgraph/page/api.js:1178]: Didn't see any weird behavior going on. Not sure how else I could make extra checks.

**Pull request merged**